common/testdata/romana.sample.yaml

```diff
@@ -14,7 +12,7 @@ services:
     config:
       store: 
         type: sqlite3
-        database: ./ipam.sqlite3
+        database: /tmp/ipam.sqlite3
```

Comment:
Ok, but it's no big issue for us to put it into /var/tmp and it gives people a chance to continue even if they accidentally rebooted a machine.